app/graphql/types/input/user_recipes/update_type.rb

```diff
@@ -0,0 +1,10 @@
+module Types
+  module Input
+    module UserRecipes
+      class UpdateType < Types::BaseInputObject
+        argument :id, Float, required: true
```

Comment:
Is `Float` what we want for the argument?

Reply:
Good question. I defaulted to float as our DB diagram had each individual recipe as float. Let me know what direction we want and I can change the code and tests accordingly.